api/api-compare-v1/src/test/groovy/com/thoughtworks/go/apiv1/compare/representers/JobHistoryItemRepresenterTest.groovy

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2019 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.thoughtworks.go.apiv1.compare.representers
+
+import com.thoughtworks.go.domain.JobResult
+import com.thoughtworks.go.domain.JobState
+import com.thoughtworks.go.presentation.pipelinehistory.JobHistoryItem
+import org.junit.jupiter.api.Test
+
+import static com.thoughtworks.go.api.base.JsonUtils.toObjectString
+import static net.javacrumbs.jsonunit.fluent.JsonFluentAssert.assertThatJson
+
+class JobHistoryItemRepresenterTest {
+  @Test
+  void 'should deserialize into json'() {
+    Date date = new Date();
+    def jobHistoryItem = new JobHistoryItem("jobName", JobState.Completed, JobResult.Passed, date)
+    jobHistoryItem.setId(5)
+
+    def actualJson = toObjectString({ JobHistoryItemRepresenter.toJSON(it, jobHistoryItem) })
+
+    def expectedJson = [
+      "name"          : "jobName",
+      "scheduled_date": date.getTime().toString(),
```

Comment:
Any reason for serializing long? I think it should be a string with TZ information.

Reply:
@maheshp suggested that we should prefer date in milliseconds as it is easier to be converted into appropriate time. With string with TZ info, users might have to parse the string.